### PR TITLE
scenario picker

### PR DESCRIPTION
**Pull request by michaelkirk**
Attempts to addresses this feedback:  https://github.com/a-b-street/abstreet/pull/571#issuecomment-802972514

Screen Shot 2021-03-19 at 12 01 35 PM

Screen Shot 2021-03-19 at 12 04 46 PM

### CODE REVIEW

File: game/src/sandbox/gameplay/freeform.rs [hunk 1]
```diff
@@ -253,8 +253,9 @@ impl ChangeScenario {
         }
         col.push(
             ctx.style()
-                .btn_outline
-                .text("Import your own data")
+                .btn_plain
+                .btn()
+                .label_underlined_text("Learn how to import your own data.")
```
Comment (michaelkirk) [inline game/src/sandbox/gameplay/freeform.rs:258]: Unrelated change - we should style things that open the browser as links.

**Comment by dabreegster** [comment]
Thanks -- this looks more button-like!
